internal/telemetry/metrics/const.go

```diff
@@ -34,10 +34,10 @@ var (
 // DefaultViews are a set of default views to view HTTP and GRPC metrics.
 var (
 	DefaultViews = [][]*view.View{
-		GRPCServerViews,
```

Comment:
We were double registering `GRPCServerViews` 👀

Reply:
Nice catch